recipes/libnetcdf/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ $(uname) == Darwin ]; then
+    brew remove --force $(brew list)
+fi
+
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+./configure \
+    --enable-shared \
+    --enable-netcdf-4 \
+    --enable-dap \
+    --without-ssl \
```

Comment:
@kmuehlbauer is correct. That was in the original recipe, but it is not valid. (Also the two options below.)